Gemfile.2.7.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    zendesk_api (3.0.5)
+    zendesk_api (3.0.6)
```

Comment:
It appears to me, the tests were running the old version!!

This is true for all lock files. I think the automation is not working. `!!!`